packages/reporter/cypress/integration/suites_spec.ts

```diff
@@ -31,99 +31,80 @@ describe('suites', () => {
   })
 
   it('includes the class "suite"', () => {
-    cy.contains('suite 1')
-    .closest('.runnable')
-    .should('have.class', 'suite')
+    cy.contains('suite 1').closest('.runnable').should('have.class', 'suite')
 
     // ensure the page is loaded before taking snapshot
     cy.contains('test 4').should('be.visible')
     cy.percySnapshot()
   })
 
   it('includes the state as a class', () => {
-    cy.contains('suite 1')
-    .closest('.runnable')
-    .should('have.class', 'runnable-failed')
+    cy.contains('suite 1').closest('.runnable').should('have.class', 'runnable-failed')
 
-    cy.contains('suite 2')
-    .closest('.runnable')
-    .should('have.class', 'runnable-passed')
+    cy.contains('suite 2').closest('.runnable').should('have.class', 'runnable-passed')
   })
 
   describe('expand and collapse', () => {
     it('is expanded by default', () => {
       cy.contains('suite 1')
-      .parents('.collapsible').as('suiteWrapper')
+      .parents('.collapsible')
+      .as('suiteWrapper')
       .should('have.class', 'is-open')
-      .find('.collapsible-content').eq(0)
+      .find('.collapsible-content')
+      .eq(0)
```

Comment:
I don't understand why one test allows inline and the other requires indention. Is it reaching a max length? Sorry, I'm completely ignorant to prettier.